src/main/webapp/administration.jsp

```diff
@@ -7,7 +8,13 @@
 <%@ include file="functions.jsp" %>
 
 
-<%    Language.setLanguage(request);
+<%
+    int benutzerID = ((Integer) session.getAttribute("BenutzerID")).intValue();
+    Benutzer benutzer = BenutzerDB.getById(benutzerID);
+    boolean isAdmin = benutzer.isAdmin();
+    if (isAdmin) {
```

Comment:
This change should be applied to multiple admin pages. In fact, it would be more efficient to handle this in the AbstractBackendServlet. There is already a function called `isLoginRequired()` that can be overridden by inherited classes. I think it would be a good idea to add a similar function there which is called `isAdminRequired()` that can be overridden by all Backend Servlet implementations. Then we can handle it in a similar way inside the initRequest function, and it should be the least overall effort.

Reply:
done look at commit: isAdminRequired function